dlme_airflow/drivers/iiif_json.py

```diff
@@ -54,15 +54,24 @@ def _extract_specified_fields(self, iiif_manifest: dict) -> dict:
                 if (
                     len(result) == 1
                 ):  # the JSONPath expression found exactly one result in the manifest
-                    output[name] = result[0].strip()
+                    output[name] = __class__._stringify_and_strip_if_list(result[0])
                 else:  # the JSONPath expression found exactly one result in the manifest
                     if name not in output:
                         output[name] = []
 
                     for data in result:
-                        output[name].append(data.strip())
+                        output[name].append(
+                            __class__._stringify_and_strip_if_list(data)
+                        )
         return output
 
+    @classmethod
+    def _stringify_and_strip_if_list(cls, possible_list) -> list[str]:
+        if isinstance(possible_list, list):
+            return [str(elt).strip() for elt in possible_list]
+        else:
+            return possible_list
+
```

Comment:
Is there any reason not to make this a straight up function rather than a class method?

Reply:
nope!  i initially wrote it as a regular method, but then turned it into a class method since it doesn't access any ivars.  but it does seem generally useful enough to just be a straight up function?

where would you put it as a regular function?  this module, but outside the class??  somewhere in `utils`?

Reply:
It could just go at the bottom of `iiif_json.py` or somewhere in `utils` I guess if it might be generally useful.

Reply:
https://github.com/sul-dlss/dlme-airflow/pull/224